bluesky/tests/test_metadata.py

```diff
@@ -0,0 +1,14 @@
+import bluesky
+import ophyd
+from bluesky import RunEngine
+
+def test_blueskyversion():
+    re = RunEngine()
+    print(re.md.get('BLUESKY_VERSION'))
```

Comment:
It's usually better to remove debug prints from tests once they are working, just to avoid noise when someone needs to run `pytest -s` to debug another test.

Reply:
updated